server/graphql/v1/CollectiveInterface.js

```diff
@@ -1174,6 +1174,11 @@ const CollectiveFields = () => {
         return collective.isHost();
       },
     },
+    isTrustedHost: {
+      type: new GraphQLNonNull(GraphQLBoolean),
+      description: 'Returns whether this host is trusted or not',
+      resolve: collective => get(collective, 'data.isTrustedHost', false),
```

Comment:
:ice_cream: `data` is unstructured, if someone sets `data.isTrustedHost` to `null` we'll have a crash here. It's safer to do:

```suggestion
      resolve: collective => Boolean(get(collective, 'data.isTrustedHost')),
```

Reply:
Done. 👍🏽